### PR TITLE
[15.0] [MIG] stock_location_analytic_mrp

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Depends on:

- https://github.com/ForgeFlow/eficent-odoo-addons/pull/294

### CODE REVIEW

File: stock_location_analytic_mrp/views/mrp.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!-- Deactive duplicate view for field analytic_account_id -->
```
Comment (AaronHForgeFlow) [inline stock_location_analytic_mrp/views/mrp.xml:3]: Why it is duplicated? if the duplicated view is in our modules perhaps we can update that view instead.

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM

File: stock_location_analytic_mrp/views/mrp.xml (hunk 1 above)
Comment (JasminSForgeFlow) [reply stock_location_analytic_mrp/views/mrp.xml:3]: one is OCA module and one is from base module so we can't change anything there I think

Comment (AaronHForgeFlow) [reply stock_location_analytic_mrp/views/mrp.xml:3]: ok! LGTM can you just rebase to see if CI is green before merging?

Comment (JasminSForgeFlow) [reply stock_location_analytic_mrp/views/mrp.xml:3]: it depends on https://github.com/ForgeFlow/eficent-odoo-addons/pull/294, so need to merge first it

Comment (AaronHForgeFlow) [reply stock_location_analytic_mrp/views/mrp.xml:3]: yes, my bad, sorry

Comment (AaronHForgeFlow) [reply stock_location_analytic_mrp/views/mrp.xml:3]: Just merged the dependency :)

**Pull request merged**